src/lib/theme/index.tsx

```diff
@@ -40,16 +41,19 @@ export const lightTheme: Colors = {
   currentColor: 'currentColor',
 }
 
+const darkThemeAccent = '#2172E5'
+
 export const darkTheme: Colors = {
   // surface
-  accent: '#2172E5',
+  accent: darkThemeAccent,
   container: '#191B1F',
   module: '#2C2F36',
   interactive: '#40444F',
   outline: '#565A69',
   dialog: '#000000',
 
   // text
+  onAccentText: readableColor(darkThemeAccent),
```

Comment:
This should be onAccent, not onAccentText.
I'm more concerned with consistency, as this will be visible to external consumers.

Reply:
are you sure `on` is clear? it sounds like a handler, and even in the context of a theme definition I find the name v confusing. including `text` at least makes it clear how the value will be applied.